example_site/src/components/Collections.js

```diff
@@ -117,27 +137,37 @@ export default function Collections() {
         filtered by length and/or aspect ratio. Show any or all of
         them any way that you like."
       />
-      {isLoading && <div className="loading">Loading...</div>}
-      {isError && <div className="error">Error: {error}</div>}
-      {isSuccess && (
-        <>
+      {isLoading ? (
+        <div className="loading">Loading...</div>
+      ) : null}
+
+      {isError ? (
+        <div className="error">Error: {error}</div>
+      ) : null}
+
+      {isSuccess ? (
+        <>{/*
           <h3>Template Filters</h3>
-          <JsonEditor value={templateFilter} onChange={setTemplateFilter} />
-          <ul className="collections">
-            {collections.map((collection) => (
-              <Collection
-                key={collection.id}
-                collection={collection}
+        <JsonEditor value={templateFilter} onChange={setTemplateFilter} /> */}
+          <div className="browser-columns">
+            <div className="categories">
+              <h2>Categories</h2>
+              {collections.map((collection) => (
+                <CollectionNames
+                collection ={collection}
                 setSelectedCollection={setSelectedCollection}
-                expand={Boolean(
-                  selectedCollection && selectedCollection.id === collection.id
-                )}
-                templateFilter={templateFilter}
+                />
+              ))}
+            </div>
+            <div className="templates">
+              <CollectionTemplates
+                collection ={selectedCollection}
+                expand={selectedCollection != null}
```

Comment:
I don't think we need to pass down this `expand` prop anymore.

`CollectionTemplates` handles its own loading/error state, and the `expand` value is just used to render the HTML or not, so you could rewrite this like so:
```
{selectedCollection ? (
    <CollectionTemplates .../>
: null)
```